### PR TITLE
Bump lighthouse to 7.4.0 and others

### PR DESCRIPTION
**Pull request by erikxiv**
Update dependencies, in particular lighthouse

Signed-off-by: Erik Larsson <erik.larsson@schibsted.com>

### CODE REVIEW

**Review by kaimallea — APPROVED** [review]

**Comment by kaimallea** [comment]
@erikxiv thanks for contributing this change!

**Comment by erikxiv** [comment]
Now I feel guilty for seemingly breaking the build 😱. On my end, the tests were flaky, diifferent errors when I re-ran the tests  and I blamed my setup as the situation was the same without my changes. Looked to me like concurrency issues with the database, where maybe two tests ran simultaneous with the same database? On the other hand the commit history looks stable, so might be that one of the dependency bumps caused a difference in how the tests are run (the functionality seems to work fine as far as I could see).

TL;DR; I will not get offended if you revert my contribution to fix master. Or try to rerun the tests.

**Comment by kaimallea** [comment]
@erikxiv whoops! I didn't realize it was broken, I'm pretty sure I saw all tests passing prior to merging. 🤔 Will need to investigate re: dependencies. In the meantime, I will check out your PRs with fixes, much appreciated!
